internal/controllers/phases.go

```diff
@@ -367,7 +358,7 @@ func (p *phaseReconciler) install(ctx context.Context) (reconcile.Result, error)
 
 	log.Info("Installing provider")
 
-	if _, err := installer.Install(cluster.InstallOptions{}); err != nil {
+	if err := clusterClient.ProviderComponents().Create(p.components.Objs()); err != nil {
```

Comment:
Look like we don't use `installer` anymore. Should we create an instance of it on line 356?